### PR TITLE
New branding

### PR DESCRIPTION
**Pull request by mradkov**
supersedes #892 

thanks @maxkpower

### CODE REVIEW

**Review by davidyuk — COMMENTED** [review]
Proposing to merge #894 before and rebase it on top of that

File: src/assets/soundWave.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...h="2" height="20" transform="translate(880 546)" fill="url(#a)"/><rect width="2" height="22" transform="translate(880 544)" fill="url(#a)"/><rect width="2" height="23" transform="translate(883 543)" fill="url(#a)"/><rect width="2" height="22" transform="translate(886 544)" fill="url(#a)"/><rect width="2" height="23" transform="translate(889 543)" fill="url(#a)"/><rect width="2" height="20" transform="translate(892 546)" fill="url(#a)"/><rect width="2" height="23" transform="translate(895 543)" fill="url(#a)"/><rect width="2" height="20" transform="translate(898 546)" fill="url(#a)"/><rect width="2" height="20" transform="translate(901 546)" fill="url(#a)"/><rect width="2" height="28" transform="translate(904 538)" fill="url(#a)"/><rect width="2" height="20" transform="translate(907 546)" fill="url(#a)"/><rect width="2" height="22" transform="translate(907 544)" fill="url(#a)"/><rect width="2" height="22" transform="translate(910 544)" fill="url(#a)"/><rect width="2" height="23" transform="translate(913 543)" fill="url(#a)"/><rect width="2" height="22" transform="translate(916 544)" fill="url(#a)"/><rect width="2" height="23" transform="translate(919 543)" fill="url(#a)"/></g></svg>
```
[the remark this answers is not in the document]
Comment (mradkov) [reply src/assets/soundWave.svg:1]: > Proposing to merge #894 before and rebase it on top of that

Done that. Thanks.

File: src/views/Maintenance.vue [hunk 2]
```diff
@@ -48,7 +51,7 @@ export default {
     line-height: 55px;
     text-align: center;
     margin: 50px 1rem;
-    color: #3290e2;
+    color: blue;
```
[the remark this answers is not in the document]
Comment (mradkov) [reply src/views/Maintenance.vue:52]: cc @maxkpower ?

**Comment by venimus** [comment]
#894 already includes the fonts

**Comment by mradkov** [comment]
> #894 already includes the fonts

Yes. This PR is rebased on top and I've dropped the added font files here in favor of #894.

File: src/views/Maintenance.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (mradkov) [reply src/views/Maintenance.vue:52]: So it turns out we should use the superhero blue. Which is `#1161fe`.

[the remark this answers is not in the document]
Comment (maxkpower) [reply src/views/Maintenance.vue:52]: #1161fe

File: src/components/layout/RightSectionTopics.vue [hunk 3]
```diff
@@ -7,7 +7,6 @@
       v-bind="$attrs"
       :closed="closed"
     >
-      <img src="../../assets/iconTrending.svg">
```
[the remark this answers is not in the document]
Comment (mradkov) [reply src/components/layout/RightSectionTopics.vue:10]: Fixed in: https://github.com/aeternity/superhero-ui/pull/893/commits/4eb07eef331ecb86dcb18abd67b8bef6bb515844

**Pull request closed without merging**